GUIDELINES.md

```diff
@@ -191,9 +191,9 @@ complete the following steps.
 ### Pages
 
 The _[Pages](https://jekyllrb.com/docs/pages/)_ are located at the
-[pages](/pages/) directory, to create one follow these steps:
+[pages](.) directory, to create one follow these steps:
```

Comment:
```suggestion
[root](.) of the repository, to create one follow these steps:
```

Reply:
Ah it refers to those pages. Part of those seem to conflict with *.html files in /include etc, so its a not super clear. OK, let's have it with your suggestion.